index.js

```diff
@@ -282,6 +282,9 @@ function notifyReady (done) {
       var IS_TEST = process.env.NODE_ENV === 'test'
       if (IS_TEST) win.setSize(1000, 800, false)
       if (argv.debug) win.webContents.openDevTools()
+
+      win.maximize()
+      win.setFullScreen(true)
```

Comment:
I would avoid to call win.setFullScreen(true)
it sets the app...fullscreen (here on ubuntu) and I don't think is the desired behaviour

Reply:
You're right good catch! thanks!